_security-plugin/access-control/multi-tenancy.md

```diff
@@ -7,7 +7,16 @@ nav_order: 30
 
 # OpenSearch Dashboards multi-tenancy
 
-*Tenants* in OpenSearch Dashboards are spaces for saving index patterns, visualizations, dashboards, and other OpenSearch Dashboards objects. By default, all OpenSearch Dashboards users have access to two tenants: **Private** and **Global**. The global tenant is shared between every OpenSearch Dashboards user. The private tenant is exclusive to each user and can't be shared.
+*Tenants* in OpenSearch Dashboards are spaces for saving index patterns, visualizations, dashboards, and other OpenSearch Dashboards objects. By default, all OpenSearch Dashboards users have access to two independent tenants:
+
+- **Private -** This tenant is exclusive to each user and can't be shared. It can't be used to access routes or index patterns made by the user's global tenant.
```

Comment:
One small nit - maybe "You can't use it" rather than "It can't be used" (active voice)